### PR TITLE
fix(public-doscite-v9): global styles should not be applied to story elements

### LINKED ISSUE #32471: [Bug]: Global Storybook styles are being applied to story elements
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
.
```

### Are you reporting an Accessibility issue?

no

### Reproduction

https://react.fluentui.dev/?path=/docs/components-drawer--docs#separator

### Bug Description

Some elements inside the stories examples are affected by global CSS used to style main global Storybook elements. This was not the case for Storybook 6 since it uses iframes. Now with Storybook 7, some styles are broken.

For example, global styles applied to h2 titles are affecting stories that contain a v9 component that renders a h2 tag:

Current:
image

Expected:
image

For this specific issue, the culprit is the selector `#storybook-docs .sbdocs h2`, but our `docs-root.css` file has several global styles that applies to all storybook examples.

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by dmytrokirpa**
## Previous Behavior

The global styles of Storybook for specific native elements have unintentionally impacted the stories.

## New Behavior

Refine global CSS selectors for elements to enhance specificity and prevent style leakage into stories.

## Related Issue(s)

- Fixes #32471

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: .storybook/docs-root.css [hunk 1]
```diff
@@ -20,7 +20,7 @@
   z-index: 99;
```
Comment (fabricteam) [inline .storybook/docs-root.css:20]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32472&commitId=430bb44162db622b88a42620b576edfcd4553af5&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (fabricteam) [inline .storybook/docs-root.css:20]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32472&commitId=430bb44162db622b88a42620b576edfcd4553af5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|4| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1147| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2251| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/BaseButton-mount_base.html">662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/BaseButton-mount.html">679</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Breadcrumb-mount_base.html">1850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Breadcrumb-mount.html">1850</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Checkbox-mount_base.html">1711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Checkbox-mount.html">1761</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/CheckboxBase-mount_base.html">1499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/CheckboxBase-mount.html">1526</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ChoiceGroup-mount_base.html">3037</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ChoiceGroup-mount.html">3018</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ComboBox-mount_base.html">689</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ComboBox-mount.html">691</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/CommandBar-mount_base.html">6924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/CommandBar-mount.html">6949</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ContextualMenu-mount_base.html">15965</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ContextualMenu-mount.html">16361</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DefaultButton-mount_base.html">830</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DefaultButton-mount.html">799</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRow-mount_base.html">2226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRow-mount.html">2309</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRowFast-mount_base.html">2257</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRowFast-mount.html">2267</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRowNoStyles-mount_base.html">2062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DetailsRowNoStyles-mount.html">2124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Dialog-mount_base.html">2720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Dialog-mount.html">2867</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DocumentCardTitle-mount_base.html">238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/DocumentCardTitle-mount.html">238</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Dropdown-mount_base.html">2054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Dropdown-mount.html">2053</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/FocusTrapZone-mount_base.html">1167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/FocusTrapZone-mount.html">1163</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/FocusZone-mount_base.html">1091</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/FocusZone-mount.html">1109</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-mount_base.html">43078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-mount.html">43181</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-virtual-rerender_base.html">20590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-virtual-rerender.html">20716</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">53045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedList-virtual-rerender-with-unmount.html">53770</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-mount_base.html">241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-mount.html">239</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-virtual-rerender_base.html">221</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-virtual-rerender.html">234</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">235</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/IconButton-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/IconButton-mount.html">1161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Label-mount_base.html">342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Label-mount.html">346</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Layer-mount_base.html">2835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Layer-mount.html">2757</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Link-mount_base.html">419</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Link-mount.html">409</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/MenuButton-mount_base.html">1040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/MenuButton-mount.html">974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/MessageBar-mount_base.html">22263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/MessageBar-mount.html">22474</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Nav-mount_base.html">2057</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Nav-mount.html">2040</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/OverflowSet-mount_base.html">801</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/OverflowSet-mount.html">802</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Panel-mount_base.html">1911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Panel-mount.html">1900</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Persona-mount_base.html">749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Persona-mount.html">781</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Pivot-mount_base.html">905</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Pivot-mount.html">928</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/PrimaryButton-mount_base.html">937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/PrimaryButton-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Rating-mount_base.html">4797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Rating-mount.html">4671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SearchBox-mount_base.html">915</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SearchBox-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Shimmer-mount_base.html">1947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Shimmer-mount.html">1984</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Slider-mount_base.html">1375</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Slider-mount.html">1373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SpinButton-mount_base.html">2971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SpinButton-mount.html">3033</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Spinner-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Spinner-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SplitButton-mount_base.html">1863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SplitButton-mount.html">1928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Stack-mount_base.html">424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Stack-mount.html">427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/StackWithIntrinsicChildren-mount_base.html">909</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/StackWithIntrinsicChildren-mount.html">883</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/StackWithTextChildren-mount_base.html">2841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/StackWithTextChildren-mount.html">2801</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SwatchColorPicker-mount_base.html">6612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/SwatchColorPicker-mount.html">6588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/TagPicker-mount_base.html">1485</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/TagPicker-mount.html">1510</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Text-mount_base.html">403</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Text-mount.html">398</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/TextField-mount_base.html">951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/TextField-mount.html">957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-mount_base.html">866</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-mount.html">882</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-virtual-rerender_base.html">596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-virtual-rerender.html">593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1309</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1285</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Toggle-mount_base.html">646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/Toggle-mount.html">636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/buttonNative-mount_base.html">197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32472/perf-test/buttonNative-mount.html">207</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: .storybook/docs-root.css [hunk 2]
```diff
@@ -45,13 +45,13 @@
   border-radius: 24px;
 }
 
-#storybook-docs .sbdocs:not(.sbdocs-preview) hr {
+#storybook-docs .sbdocs hr:not(.sbdocs-preview hr) {
```
Comment (Hotell) [inline .storybook/docs-root.css:48]: these negations add even more complexity, can we  aggregate and simplify all of these ?

eg
```css
#storybook-docs .sbdocs:not(.sbdocs-preview) p {}

#storybook-docs .sbdocs:not(.sbdocs-preview) hr {}

#storybook-docs .sbdocs:not(.sbdocs-preview) h2 {}

#storybook-docs .sbdocs:not(.sbdocs-preview) h2 code {}
```

File: .storybook/docs-root.css [hunk 3]
```diff
@@ -426,7 +426,7 @@ h1.fluent {
   letter-spacing: -0.16px;
 }
 
-h1 .fluent-version {
+h1.fluent .fluent-version {
```
Comment (Hotell) [inline .storybook/docs-root.css:429]: can you pls point out where is this `.fluent` or `.fluent-version` class being used ? I'm unable to find it in docs = if it's not present lets remove it.

Comment (dmytrokirpa) [reply .storybook/docs-root.css:429]: The `.fluent` and `.fluent-version` classes are utilized within our codebase.

For example, you can see their usage here: https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/Concepts/Introduction.stories.mdx?plain=1#L9

File: .storybook/docs-root.css (hunk 2 above)
Comment (dmytrokirpa) [reply .storybook/docs-root.css:48]: I tried that, and unfortunately it doesn't work as expected as there multiple elements with the `sbdocs` class nested. Here's an example:

image

File: .storybook/docs-root.css [hunk 4]
```diff
@@ -15,12 +15,12 @@
   margin-top: 49px;
```
Comment (fabricteam) [inline .storybook/docs-root.css:15]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32472&commitId=790627581ff4990dd0d6efdbd42c85c5f20ae9b2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: .storybook/docs-root.css (hunk 4 above)
Comment (fabricteam) [inline .storybook/docs-root.css:15]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32472&commitId=9de21a0301451749fb215acce71b40598879fc28&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: .storybook/docs-root.css (hunk 2 above)
Comment (Hotell) [reply .storybook/docs-root.css:48]: sad to see this. the tax of tweaking non public APIs... ty for clarification

**Review by Hotell — APPROVED** [review]

**Review by marcosmoura — APPROVED** [review]

**Pull request merged**